lib/install/management/delete.go

```diff
@@ -85,6 +84,10 @@ func (d *Dispatcher) DeleteVCH(conf *metadata.VirtualContainerHostConfigSpec) er
 	if err = d.destroyResourcePoolIfEmpty(conf); err != nil {
 		log.Warnf("VCH resource pool is not removed, %s", err)
 	}
+
+	log.Infoln("Removing volume stores...")
```

Comment:
You cannot remove volume after the appliance vm is removed. Cause we need the capability to rerun delete if anything wrong, and appliance vm keeps all the information for next run.